project/npda/templates/npda/npdauser_form.html

```diff
@@ -20,6 +20,12 @@
                 </select>
               {% elif field.field.widget|is_textinput %}
                 <input type="text" id="{{ field.id_for_label }}" name="{{ field.html_name }}" {% if field.value %} value="{{ field.value }}" {% endif %} class="input rcpch-input-text">
+              {% elif field.field.widget|is_checkbox and request.user.is_superuser and field.id_for_label == 'id_is_superuser' or field.id_for_label == 'id_is_staff' %}
```

Comment:
@eatyourpeas completely see what you're doing here Simon and is exaxtly what the form needs. Doesn't look like it's working though - for my user of type Coordinator (who does not have is_staff or is_superuser ticked) they are able to see all 4 checkboxes and change them as they wish